test/import.service.test.js

```diff
@@ -0,0 +1,62 @@
+const supertest = require('supertest');
+const { buildConfig } = require('../src/util/config');
+const { initialize } = require('../src/server/server');
+const validParam = require('./fixtures/parametersObjs/paramWithExport');
+const paramNoExport = require('./fixtures/parametersObjs/paramNoExport.json');
+const testParamTwoExports = require('./fixtures/parametersObjs/paramTwoExports.json');
+const testParamNoValString = require('./fixtures/parametersObjs/paramNoValueString.json');
+const { SINGLE_AGENT_PROVENANCE } = require('./fixtures/testProvenanceUtils');
+const { client } = require('../src/util/mongo');
+const { cleanUpDb } = require('./populateTestData');
+
+const config = buildConfig();
+const server = initialize(config);
+
+describe('Testing $import with no specified measure bundle', () => {
+  beforeEach(async () => {
+    await client.connect();
+  });
+  test.skip('Returns 202 on Valid Request', async () => {
```

Comment:
```suggestion
  test('Returns 202 on Valid Request', async () => {
```

Reply:
This is actually skipped on purpose for now due to jest race conditions. I'll add a comment!